collect_app/build.gradle

```diff
@@ -201,6 +201,7 @@ dependencies {
     implementation 'androidx.browser:browser:1.2.0'
     implementation 'androidx.appcompat:appcompat:1.1.0'
     implementation "androidx.work:work-runtime:2.3.4"
+    implementation 'com.google.android.material:material:1.3.0-alpha01'
```

Comment:
We shouldn't be using an alpha or beta version. Looking at https://maven.google.com/web/index.html it looks like the stable is currently 1.1.0 and they have a beta of 1.20 and an alpha of 1.3.0 out 😒